### PR TITLE
Hw01

### PR DESCRIPTION
**Pull request by samruddhis**
@tylerallan @njie-DP

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,8 +12,8 @@ There are currently two branches:
 
 This is where I practice & submit code for the course [Programming with Data](https://progdata.netlify.app/#about)!
 
-# About the Student
+# About the Student :star:
```
Comment (tylerallan) [inline README.md:15]: Love these little additions

File: hw01/hw01_exploring_data.Rmd [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+---
+
+In this document, we will explore the the palmerpenguins dataset.
+
+```{r}
+library(palmerpenguins)
+
+print(penguins)
+```
+
```
Comment (tylerallan) [inline hw01/hw01_exploring_data.Rmd:40]: You may need to add in a couple more functions here other than print. Things like nrow or ncolumn

**Review by tylerallan — COMMENTED** [review]
Great work!

File: hw01/hw01_exploring_data.Rmd (hunk 2 above)
Comment (tylerallan) [reply hw01/hw01_exploring_data.Rmd:40]: Ignore I saw you added these after I started reviewing.

File: hw01/hw01_exploring_data.Rmd [hunk 3]
```diff
@@ -0,0 +1,38 @@
+---
+title: "hw01_exploring_data"
+author: "Samruddhi Shinde"
+date: "1/24/2021"
+output: html_document
```
Comment (tylerallan) [inline hw01/hw01_exploring_data.Rmd]: make sure to add in github_document: default here

Comment (njie-DP) [reply hw01/hw01_exploring_data.Rmd]: As Tyler said, I did not find the file extension ended with .md.

**Review by njie-DP — COMMENTED** [review]
Great job, overall! Your introduction page looks really nice. It would be helpful if you could clean up the files under this branch. I saw the .gitignore, .SD_Store, and a navigating file which I am not sure whether they are necessary here.

**Pull request merged**